create_aio_app/template/{{cookiecutter.project_name}}/Makefile

```diff
@@ -43,7 +43,14 @@ migrations:
 
 migrate:
 	@docker exec -it {{ cookiecutter.project_name }} alembic -n alembic:dev upgrade head;
+
+wait_postgres:
+	python ./{{ cookiecutter.project_name }}/utils.py
```

Comment:
u should use a specific version of python `python3`

Reply:
Personally I prefer `python3 -m {{ cookiecutter.project_name }}.utils`